<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StatePropertiesDlg</class>
 <widget class="QDialog" name="StatePropertiesDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>314</width>
    <height>410</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>State Properties</string>
  </property>
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="margin" stdset="0">
    <number>11</number>
   </property>
   <item>
    <layout class="QVBoxLayout">
     <item>
      <layout class="QGridLayout">
       <item row="6" column="1">
        <widget class="QLineEdit" name="le_linewidth">
         <property name="text">
          <string>1</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="le_entry"/>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="le_code"/>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="le_name"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="TextLabel2">
         <property name="text">
          <string>Code:</string>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QPushButton" name="pb_color">
         <property name="text">
          <string>Colour</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="TextLabel1">
         <property name="text">
          <string>Name:</string>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="le_radius">
         <property name="text">
          <string>40</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="textLabel1">
         <property name="text">
          <string>Moore Outputs:</string>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="le_mooreoutputs"/>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="TextLabel4">
         <property name="text">
          <string>Line width:</string>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QLabel" name="lb_colorpreview">
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="text">
          <string notr="true"/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::TextSelectableByMouse</set>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="TextLabel3">
         <property name="text">
          <string>Radius:</string>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="le_exit"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Entry Actions:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Exit Actions:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="TextLabel1_2">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="text">
        <string>Description:</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="te_description">
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="margin" stdset="0">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="pb_ok">
         <property name="text">
          <string>OK</string>
         </property>
         <property name="default">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pb_cancel">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>le_name</tabstop>
  <tabstop>le_code</tabstop>
  <tabstop>le_mooreoutputs</tabstop>
  <tabstop>le_entry</tabstop>
  <tabstop>le_exit</tabstop>
  <tabstop>le_radius</tabstop>
  <tabstop>le_linewidth</tabstop>
  <tabstop>pb_color</tabstop>
  <tabstop>pb_ok</tabstop>
  <tabstop>pb_cancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
